client/src/components/presentational/DriverPopup.tsx

```diff
@@ -101,16 +105,13 @@ function DriverPopup({ trip, setDriverStatus }:
   };
 
   const handleClickSubmitButton = async() => {
-    const payload = { response: 'confirm', riderId: trip.rider.id, tripId: trip.id };
-    const data = await sendDriverResponse(client, dispatch, payload);
-    if (data === MATCHING_SUCCESS) {
+    const { data } = await notifyDriverResponse();
+    if (data.sendResponse === MATCHING_SUCCESS) {
       dispatch(setTrip({ id: trip.id }));
       dispatch(setRider({ id: trip.rider.id }));
-      await getPickUpPos(client, dispatch, { id: trip.id });
       return setDriverStatus(DRIVER_MATCHING_SUCCESS);
-    } else {
-      showAlert(data.result);
     }
+    showAlert(data.sendResponse.result);
```

Comment:
이 부분 제가 수정하면서 놓친부분이라서
```suggestion
    showAlert(data.sendResponse);
```
이렇게 해주시면 될 것 같습니다!

Reply:
네 수정하겠습니다!